py/picca/delta_extraction/expected_fluxes/true_continuum.py

```diff
@@ -0,0 +1,486 @@
+"""This module defines the class Dr16ExpectedFlux"""
+import logging
+import multiprocessing
+
+import fitsio
+from astropy.io import fits
+import numpy as np
+from scipy.interpolate import interp1d
+import healpy
+from pkg_resources import resource_filename
+
+from picca.delta_extraction.astronomical_objects.forest import Forest
+from picca.delta_extraction.astronomical_objects.pk1d_forest import Pk1dForest
+from picca.delta_extraction.errors import ExpectedFluxError
+from picca.delta_extraction.expected_flux import ExpectedFlux
+
+accepted_options = ["input directory", "iter out prefix",
+                    "num processors",
+                    "var lss binning"]
+
+defaults = {
+    "iter out prefix": "delta_attributes",
+}
+
+
+class TrueContinuum(ExpectedFlux):
+    """Class to the expected flux calculated used the true continuum unabsorbed contiuum
+    for mocks. It uses var_lss pre-computed from mocks and the mean flux estimated from 
+
+    Methods
+    -------
+    extract_deltas (from ExpectedFlux)
+    __init__
+    _initialize_arrays_lin
+    _initialize_arrays_log
+    _parse_config
+    compute_delta_stack
+    compute_expected_flux
+    read_true_continuum
+    read_var_lss
+    populate_los_ids
+
+    Attributes
+    ----------
+    los_ids: dict (from ExpectedFlux)
+    A dictionary to store the mean expected flux fraction, the weights, and
+    the inverse variance for each line of sight. Keys are the identifier for the
```

Comment:
@iprafols - I guess "inverse variance" here refers to the noise variance from the pipeline directly (no eta, no var_lss, etc.), right? While "weights" include the final pixel variance, including all ingredients? I think it would be good to clarify this here and elsewhere, since we use "variance" with many different meanings in picca.

Reply:
Ignasi has improved the docstring.